mappings/net/minecraft/entity/passive/PandaEntity.mapping

```diff
@@ -11,11 +11,14 @@ CLASS net/minecraft/unmapped/C_keifzdgo net/minecraft/entity/passive/PandaEntity
 	FIELD f_gkpdhaly lastRollOverAnimationProgress F
 	FIELD f_jblxaaho rollOverAnimationProgress F
 	FIELD f_jetaxnfz lieOnBackAnimationProgress F
+	FIELD f_lrhwzwjp TOTAL_ROLL_STEPS I
 	FIELD f_maffmtuv ASK_FOR_BAMBOO_TARGET Lnet/minecraft/unmapped/C_hnihytjc;
+	FIELD f_nryptvaq TOTAL_UNHAPPY_TIME I
```

Comment:
TOTAL_DISCOMFORT_TIME maybe or TOTAL_DISPLEASED_TIME? I don't know what this exactly does but I think this is the cool down between the panda rejecting bamboo I think?

Reply:
I copied this one from MM, I don't know what it does either 😅